content/tracing/advanced_usage/_index.md

```diff
@@ -830,6 +862,11 @@ public void ConfigureServices(IServiceCollection services)
 
 [dotnet opentracing]: https://www.nuget.org/packages/Datadog.Trace.OpenTracing/
 
+{{% /tab %}}
+{{% tab "C++" %}}
+
+The Datadog C++ tracer currently can only be used through the OpenTracing API. The usage instructions in this document all describe generic OpenTracing functionality.
```

Comment:
```suggestion
The Datadog C++ tracer can only be used through the OpenTracing API. The usage instructions in this document all describe generic OpenTracing functionality.
```

We don't like to imply promises of future functionality, so we try to avoid words like "currently"